windowsforms/MSIX-WindowsForms/README.md

```diff
@@ -0,0 +1,51 @@
+# MSIX Windows Forms Core Application
+
+This sample shows how to use the Windows Packaging Project to package a WinForms application running on Core.
+
+## Pre-requisites
+
+To use the Windows Packaging Project (wapproj) you need to install the Universal Windows workload in Visual Studio.
+
+To produce `MSIX` packages you must have the **Windows 10 October 2018 SDK** (aka 10.0.17763), otherwise the generated package will have the `APPX` extension.
```

Comment:
- comma after "packages"
- semicolon before "otherwise", and a comma after "otherwise"